pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -262,6 +267,29 @@ func (p *Provider) createLaunchTemplate(ctx context.Context, options *amifamily.
 	return output.LaunchTemplate, nil
 }
 
+func generatePlacement(placement *amifamily.Placement) *ec2.LaunchTemplatePlacementRequest {
+    if placement == nil {
+        return nil
+    }
+    return &ec2.LaunchTemplatePlacementRequest{
+        HostResourceGroupArn: aws.String(placement.HostResourceGroup),
+    }
+
+}
+
+func generateLicenseSpecification(licenses []string) []*ec2.LaunchTemplateLicenseConfigurationRequest {
+    result := []*ec2.LaunchTemplateLicenseConfigurationRequest{}
```

Comment:
Generally favor the `var` initialization of slices due to: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Reply:
This also means that you don't have to explicitly return nil when iterating since the var value will be `nil` by default